Apps/UnitTests/X11/App.cpp

```diff
@@ -0,0 +1,48 @@
+#define XK_MISCELLANY
+#define XK_LATIN1
+#include <X11/Xlib.h> // will include X11 which #defines None... Don't mess with order of includes.
+#include <X11/Xutil.h>
+#undef None
+#include "../Shared/Tests.cpp"
+
+namespace {
+    constexpr const char* applicationName = "Babylon Native Unit Tests";
+    constexpr const  int width = 640;
+    constexpr const int height = 480;
+    constexpr const char* wmDeleteWindowName = "WM_DELETE_WINDOW";
+}
+int main()
```

Comment:
nit: should probably have a newline here
```suggestion
}

int main()
```